src/coreWebVitals.README.md

```diff
@@ -0,0 +1,72 @@
+# Core Web Vitals
+
+Reports on Core Web Vitals using Google’s [`web-vitals`] library, and send the
+metrics to an logging endpoint when the user leaves the page.
+
+By default, only 1% of page views will the data, but it’s possible to force this
+behaviour at initialisation or asynchronously.
+
+[`web-vitals`]: https://github.com/GoogleChrome/web-vitals
+
+## Usage
+
+```js
+import { initCoreWebVitals } from '@guardian/libs';
+
+const optionalCallback = () => {
+    // An optional callback which triggers when the user leaves the page
+    log('dotcom', 'CWV payload queued for async request')
+}
+
+initCoreWebVitals({
+    browserId,
+    pageViewId,
```

Comment:
I can imagine future developers that are not part of the Dotcom team might wonder how these values can be generated/read. Is it worth making a note about this here?

Reply:
If you have access to BigQuery they can be seen as raw data there but we have a dashboard here which can be split into DCR and frontend if you go to `Filtering Section` and change `Filter Component`  https://tableau.gnmremote.com/#/views/DotcomWebVitals/Singlefilterview/guy_parr@gnm.int/397372d8-b69d-44e8-acaf-45c60de7c4ed?:display_count=n&:showVizHome=n&:origin=viz_share_link&:iid=1

Reply:
Sorry, by 'read' I meant should I read them from a cookie, from local storage, or do I need to generate new values myself? There's some assumed knowledge built in here that not everyone looking to add the ability to track their core web vitals will have. They could ask around, sure, but if we make it easier for them it might improve usage and make the world a slightly faster place.

Reply:
That’s a good point, @oliverlloyd. Actually, I think that these params should be optional, because you can still get aggregate data, even if you can’t join it to page views. I’ll also add an example of where they usually can be found and some documentation.